src/game/logic/scriptengine/scriptengine.cpp

```diff
@@ -13,56 +13,9429 @@
+        Parameter::TEAM_COMMANDBUTTON_ABILITY;
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_parameters[2] =
+        Parameter::OBJECT_TYPE;
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_numUiStrings = 4;
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_uiStrings[1] = " use ";
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_uiStrings[2] =
+        " on nearest object of type ";
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_uiStrings[3] = ".";
+
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_internalName = "TEAM_PARTIAL_USE_COMMANDBUTTON";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiName =
+        "Team_/ Use command ability -- partial -- self.";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_numParameters = 3;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_parameters[0] = Parameter::REAL;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_parameters[1] = Parameter::TEAM;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_parameters[2] = Parameter::TEAM_COMMANDBUTTON_ABILITY;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_numUiStrings = 4;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiStrings[1] = "% of ";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiStrings[2] = " perform ";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiStrings[3] = ".";
+
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_internalName =
+        "TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT";
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_uiName =
+        "Team_/ Capture unowned faction unit -- nearest.";
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_numParameters = 1;
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_parameters[0] = Parameter::TEAM;
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_numUiStrings = 2;
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_uiStrings[1] =
+        " capture the nearest unowned faction unit.";
+
+    m_actionTemplates[ScriptAction::PLAYER_CREATE_TEAM_FROM_CAPTURED_UNITS].m_internalName =
+        "PLAYER_CREATE_TEAM_FROM_C
```

Comment:
You can remove blank lines from SEP_HEAD until SEP_EOL, so it is easier to see which lines belong together in a block.